README.md

```diff
@@ -72,106 +71,85 @@ In the Fauna Cloud Console:
 
-  > NOTE: Make sure to replace `YourFaunaDBSecretHere` with the value of the secret that you copied in the previous step.
+> NOTE: The `bootstrap` script will store another environment variable in this
+> file called `FAUNADB_PUBLIC_KEY`.
 
-  > NOTE: We'll set a different variable for FAUNADB_PUBLIC_KEY in a bit, which is the important key to actually run the app!
+### 9. Bootstrap your FaunaDB collection and indexes
 
-9. Bootstrap your FaunaDB collection and indexes
+```bash
+# create database and fill with example data
+yarn bootstrap
+```
 
-  ```bash
-  node scripts/create-database.js
-  ```
+This `bootstrap` package.json script will create a Fauna database called
+`auth-example` using `scripts/create-database.js` and then fill it with
+example data using `scripts/create-example-data.js`.
 
-  The `bootstrap-db` script will output some important information. Copy down the created keys shown to you in the output. If you lose the keys, you can generate them by hand or run the bootstrap command again.
+Output should be similar to:
 
-  Output should be similar to:
+```
+Creating your FaunaDB Database...
 
-  ```
-  Creating your FaunaDB Database...
+1) Create database "auth-example"
++ Created Database "auth-example"
 
-  1) Create database "auth-example"
-  + Created Database "auth-example"
+2) Creating temporary key
++ Created Key "temp admin key for auth-example"
 
-  2) Creating temporary key
-  + Created Key "temp admin key for auth-example"
+3) Uploading Graphql Schema...
+o GraphQL schema imported
 
-  3) Uploading Graphql Schema...
-  o GraphQL schema imported
+4) Update generated User Defined Functions...
+o Updated Function "login"
+o Updated Function "logout"
+o Updated Function "me"
+o Updated Function "user_create_todo"
 
-  4) Update generated User Defined Functions...
-  o Updated Function "login"
-  o Updated Function "logout"
-  o Updated Function "me"
-  o Updated Function "user_create_todo"
+5) Create custom roles...
++ Created Role "public"
++ Created Key "Public key for auth-example"
+! Public client key: fnADmZcVSKFYEykYVRyVaQ5WN9RQ3OEmEMtweMWk
++ Created Role "user"
+-Deleted Key "temp admin key for auth-example"
 
-  5) Create custom roles...
-  + Created Role "public"
-  + Created Key "Public key for auth-example"
-  ! Public client key: fnADmZcVSKFYEykYVRyVaQ5WN9RQ3OEmEMtweMWk
-  + Created Role "user"
-  -Deleted Key "temp admin key for auth-example"
+Fauna Database schema has been created
+Claim your fauna database with "netlify addons:auth fauna"
+```
 
-  Fauna Database schema has been created
-  Claim your fauna database with "netlify addons:auth fauna"
-  ```
+### 10. Upload Graphql Schema
 
-10. Upload Graphql Schema
-
-  In the Fauna Cloud Console:
+In the Fauna Cloud Console:
 
   - Click “GraphQL” in the left navigation
   - Click “IMPORT SCHEMA”
   - Upload the schema at [`scripts/faunaSchema.graphql`](https://github.com/ptpaterson/netlify-faunadb-graphql-auth/blob/master/scripts/faunaSchema.graphql)
 
```

Comment:
It's not necessary to include the GraphQL schema import step anymore, because the script uploads the schema.

Reply:
Forgot about that!

Reply:
Updated!